core/dbt/context/configured.py

```diff
@@ -58,19 +70,21 @@ def __call__(self, var_name, default=Var._VAR_NOTSET):
             all_vars.add(my_config.vars.vars_for(lookup, adapter_type))
         all_vars.add(active_vars)
 
-        if var_name in all_vars:
-            return all_vars[var_name]
+        if not var_found and var_name in all_vars:
```

Comment:
This refactor was primarily for the happiness of our linting tools. There was no final return statement on the method in the past.